README.md

```diff
@@ -1,54 +1,57 @@
 ## static-ffmpeg
 
-Docker image with ffmpeg/ffprobe built as hardened static PIE binaries
+Docker image with [FFmpeg/FFprobe](https://ffmpeg.org) built as hardened, static PIE binaries
 with no external dependencies. Can be used with any base image even scratch.
 
-Built with
-gray,
-openssl,
-iconv,
-libxml2,
-libmp3lame,
-libtwolame,
-libfdk-aac,
-libvorbis,
-libopus,
-libmodplug,
-libtheora,
-libvpx,
-libx264,
-libx265,
-libwebp,
-libspeex,
-libvmaf
-libaom,
-libvidstab,
-libkvazaar,
-libfreetype,
-fontconfig,
-libfribidi,
-libass,
-libzimg,
-libsoxr,
-libopenjpeg,
-libdav1d,
-libxvid,
-librav1e,
-libsrt,
-libsvtav1,
-libdavs2,
-libxavs2,
-libmodplug,
-libuavs3d,
-libmysofa,
-librubberband,
-libgme,
-libopencore,
-libssh
-and all native ffmpeg codecs, formats, filters etc.
```

Comment:
Good idea, have also thought about sorting the list and maybe also sort things in the Dockerfile?

Could possibly use a table if we would like to have more info, version, description, home url? wonder if bump could be made to update the version automatically hmm